app/Http/Controllers/carController.php

```diff
@@ -38,6 +38,7 @@ public function store(Request $request)
             'description' => 'required|string|max:1000', 
             'price' => 'required|decimal:0,2|between:10000,9999999999.99',
             'image' => 'required|mimes:png,jpg,jpeg|max:2048',
+            'category_id' => 'required',
```

Comment:
it is better to check that the `category_id` exists in the `categories` table also verify it is `id`
search on how to do so

so how it can make your code better?
what if your database has categories with id from 1 -> 5

| id | category_name |
|----------|----------|
| 1 | ... |
| 2 | ... |
| 3 | ... |
| 4 | ... |
| 5 | ... |

what if i am trying to send `category_id` = `6`, I cause your code to make internal server error
and what if i send `category_id` = `hello`